README.md

```diff
@@ -20,19 +20,35 @@ https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe-support-script
 
 ### Install the `healthcheck_lite` module
 
-To install the `healthcheck_lite` module, execute the following command on the Primary Master.
+#### Using Bolt
+
+Bolt is the preffered installation and execution method and can be used either on the master or another machine with connectivity to the master. Firstly, follow the instruction for [installing Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html). PE customers are *strongly* encouraged to follow the note about directly installing the package if installing on a PE node.
+
+The following block of code will create a Boltdir under the logged in user's home directory and set up the module:
 
 ```bash
-puppet module install puppetlabs-healthcheck_lite --modulepath=/opt/puppetlabs/puppet/modules
+mkdir -p ~/Boltdir
+cd !$
```

Comment:
Cool, but just specifying `cd ~/Boltdir` makes it easier for non-Bash developers to understand "what is this code that I'm running doing on my system" which is a valid security concern for a subset of users. More so in READMEs, but also true in the tasks themselves.

Reply:
Ok, but you're raining on my Bash parade...